traffic_ops_db/test/docker/Dockerfile-db-admin

```diff
@@ -39,12 +39,10 @@ RUN yum install -y \
         rm /$(basename $TRAFFIC_OPS_RPM) && \
     yum clean all
 
-RUN /opt/traffic_ops/install/bin/install_goose.sh
```

Comment:
If you rebase, you'll find that I've basically copied these tests and modified them for the PostgreSQL Traffic Vault backend in `traffic_ops/app/db/trafficvault/test`, so those should probably get these same updates (removing `install_goose.sh`, renaming the config script, and updating `run-tvdb-test.sh`).

Reply:
Rebased to to apply these updates to the PostgreSQL Traffic Vault tests in ba14e36c69.